docs/internals/contributing/writing-code/accessibility.txt

```diff
@@ -0,0 +1,114 @@
+=============
+Accessibility
+=============
+
+All aspects of Django must be compatible with our accessibility standards,
+and with the specific support targets we aim for.
+
+.. _accessibility-standards:
+
+Accessibility standards
+=======================
+
+Django aims for conformance with `WCAG 2.2`_ at the AA level.
+WCAG is the most well established standard for web accessibility. AA-level
+guidelines are the most common legal compliance target around the world.
+
+Wherever applicable, we also attempt to follow other best practices, such as:
+
+* WCAG 2.2 AAA-level guidelines.
+* The upcoming `WCAG 3.0`_ guidelines.
+* `ATAG 2.0`_ guidelines for authoring tools.
+
+.. note::
+    Do you want to learn more about accessibility without diving straight into
+    standards? We recommend `The A11Y Project`_,
+    a community-driven effort to make digital accessibility easier.
+
+.. _WCAG 2.2: https://www.w3.org/TR/WCAG22/
+.. _WCAG 3.0: https://www.w3.org/TR/wcag-3.0/
+.. _ATAG 2.0: https://www.w3.org/TR/ATAG20/
+.. _The A11Y Project: https://www.a11yproject.com/
+
+.. _accessibility-support-targets:
+
+Accessibility support targets
+=============================
+
+Beyond standards, we want to make sure Django actually works for people using
+a wide range of input devices and assistive technologies.
+The best way to do this effectively is to take accessibility considerations
+into account as part of designing features. If in doubt, consult
+with users who rely on assistive technologies or with accessibility experts.
+
+.. _accessibility-testing-baseline:
+
+Accessibility testing baseline
+------------------------------
+
+ Always test user interface changes with:
+
+* Keyboard-only navigation
+* Automated checks in the `Accessibility Insights`_ browser extension,
+  or an equivalent tool with the `Axe`_ checker.
+
+Where appropriate, also test with:
+
+* Touch-only navigation
+* 400% browser zoom
+* forced-colors mode DevTools emulation
+
+.. note::
```

Comment:
Please try to avoid "bare" notes and use `admonition` instead. This guideline is from [Guidelines for reStructuredText files](https://docs.djangoproject.com/en/stable/internals/contributing/writing-documentation/#guidelines-for-restructuredtext-files).

Reply:
Could you say more @felixxm? I chose not to use `admonition` because to me that means the content is a "warning" of sorts. I also didn’t understand that specific guideline as from what I could see the two rendered the same as far as the published docs are concerned.

If this is about the lack of a heading (is that what "bare" means?), I chose not to because though it visually looks like a heading, it’s not properly marked up as such for screen reader users.

---

So if we want to revisit this I think I’d rather just have the content within the main body copy of the docs and avoid admonitions / notes altogether.

Reply:
If it's a warning use `warning`. Notes and admonitions are like side notes, so you theoretically don't need to read them to understand the topic. Titled admonitions are better because you don't have to read the entire note to check what it is about.

Reply:
Should now be addressed.